ws/public/main.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+(function () {
+    const canvas = document.getElementsByClassName('whiteboard')[0]
```

Comment:
please use 2 spaces for indentation across this file and everywhere else

Reply:
ok